doc/local-pv-allocation.md

```diff
@@ -0,0 +1,128 @@
+# Local PV Allocation
+The purpose of this document is to show how to provision a VM disk on a specific node to be able to run it on the same node.
+
+### Prerequesites
+You have a Kubernetes cluster up and running with CDI installed.
+
+### Create your PersistentVolume manifest
+First create a PersistentVolume yaml file with NodeAffinity field to allocate the PV on specific node you choose. Verify 
+that the label you sepcify in the NodeAffinity exists on the node you want the PV to be allocated.
+In addition, add a specific label to be matched by a PersistenVolumeClaim during the binding process.
+An example of such PersistentVolume is:
+
+```yaml
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: local-pv-allocation
+  labels:
+    node: node02
+spec:
+  storageClassName: manual
+  persistentVolumeReclaimPolicy: Delete
+  capacity:
+    storage: 10Gi
+  local:
+    path: /mnt/local-storage/local/disk2
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          - node02
+  accessModes:
+    - ReadWriteOnce
+  volumeMode: Filesystem
+
+```
+### Deploy the PersistentVolume to the cluster:
+```bash
+$ kubectl create -f manifests/example/local-pv-allocation.yaml
+```
+
+### Create your PersistenVolumeClaim manifest
+Create a PersistentVolumeClaim yaml file with a selector field that matches the lable you specified in the PersistentVolume.
+An example of such PersistentVolumeClaim is:
+
+```yaml
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: local-pv-allocation-claim
+spec:
+  storageClassName: manual
```

Comment:
storageClassName: manual #This should be the name of your local-volume storage class.